packages/appstore/src/components/elements/text/balance-text.tsx

```diff
@@ -16,7 +16,7 @@ type BalanceTextProps = {
 };
 
 const BalanceText = ({ balance, currency, size = 'm', underline_style = 'none' }: BalanceTextProps) => {
```

Comment:
Dont we need to wrap it with observer
```suggestion
const BalanceText = observer(({ balance, currency, size = 'm', underline_style = 'none' }: BalanceTextProps) => {
```